mobile_rewards/src/server.rs

```diff
@@ -371,7 +391,93 @@ impl Server {
     }
 }
 
+const CONNECT_TIMEOUT: StdDuration = StdDuration::from_secs(5);
+const RPC_TIMEOUT: StdDuration = StdDuration::from_secs(5);
+const DEFAULT_URI: &str = "http://127.0.0.1:8080";
+
 async fn update_last_follower(pool: &Pool<Postgres>, last_height: u64) -> Result {
     metrics::gauge!("reward_server_last_reward_height", last_height as f64);
-    Meta::update(pool, "last_follower_height", last_height.to_string()).await
+    MetaValue::new("last_follower_height", last_height)
```

Comment:
Is this call correct? Or should it be calling `MetaValue::update(last_height)`?

Reply:
We call insert right after, so yes this is correct

Reply:
I mean logically, are we inserting a new k:v pair for `last_follower_height:value` or are we updating an existing `last_follower_height` value here. The function name seems to imply that it should be an update?

Reply:
This inserts if the value does not exist and updates when it does, so this is what we want.

Reply:
I've fixed this so that `last_follower_height` is always a `MetaValue`, so when we want to change its value we just call `.update`

Reply:
Thanks, I think that's much cleaner :)